### DISCUSSION TITLE
Broken link on play store listing

### DISCUSSION THREAD

**Discussion by Abhishmuk** [mirrorkeydev/Appstract#40] [General]
Heya, just a simple head's up - the link to this GitHub repo on the dark theme play store page (https://play.google.com/store/apps/details?id=com.melon.appstract.dark) is broken - the final bracket is included in the url. (Great job on the icon pack btw!)

**Comment by mirrorkeydev** [comment]
I'm unable to reproduce the issue on either the mobile or browser version of the play store. Is it still broken on your end?

**Reply by Abhishmuk** [reply]
Yep, if you go to the app details on play store this is what it looks like for me 
![Screenshot_20211221-222516](https://user-images.githubusercontent.com/74917888/146999739-7b04460e-179b-4aa5-9d9b-41252f08ab35.png)

The link to GitHub has the bracket as part of the url, like [github.com)](github.com)). When clicking it shows a GitHub 404 page (of course because appstract) isn't the right name, it's appstract without the bracket). This is what it looks like - 
![Screenshot_20211221-222903](https://user-images.githubusercontent.com/74917888/147000009-442cd2fe-2f1c-4a57-be35-92850ef8bbd3.png)

It's not a major issue fortunately.

**Reply by Abhishmuk** [reply]
Huh, I checked it on the website and there  "Appstract's GitHub" itself is a hyperlink which resolves correctly, and the url is just plain text and not a link like it is on the Play Store app. Weird.

**Reply by mirrorkeydev** [reply]
Ah I see -- I was checking the link at the bottom of the `About` section, instead of the release notes section. I've updated the release notes for both Appstracts and removed the parenthesis. This might break the hyperlink for the web version, but I suspect 90%+ of users are going to be reading the release notes in the app, so it's probably more important to fix it there. Let me know if that did it for you, and thanks for bringing it to my attention!

**Reply by mirrorkeydev** [reply]
Frustratingly (and not that surprisingly), the app play store and web play store handle links in the listing description and release notes differently.

Also, I just realized your screenshot shows the release notes for v4.0.5. I just released v4.0.6 about an hour ago, so it might take a couple hours to show up on your end :)
![image](https://user-images.githubusercontent.com/35010111/147003741-fef005ed-6dff-4cfc-b40c-9add0993514b.png)
